README.md

```diff
@@ -4,6 +4,50 @@
 
 Application which runs KBC transformations
 
+## Options
+
+- `authorization` object (required): [workspace credentials](https://developers.keboola.com/extend/common-interface/folders/#exchanging-data-via-workspace)
+- `parameters`
+    - `blocks` array (required): list of blocks
+        - `name` string (optional): name of the block
+        - `codes` array (required): list of codes
+            - `name` string (optional): name of the code
```

Comment:
Isto je name optional? Nespadne to tu? Asi by to chcelo upravit toto readme aj config definition.
https://github.com/keboola/snowflake-transformation/blob/5bec787c26a04bcf5d2101aed608743232918def/src/SnowflakeTransformation.php#L111